platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/binding/api/impl/ValidatorImpl.java

```diff
@@ -93,8 +94,9 @@ public void validateRelayState(String relayState) {
         if (relayState == null) {
             throw new IllegalArgumentException("Missing RelayState on IdP request.");
         }
-        if (relayState.getBytes().length < 0 || relayState.getBytes().length > 80) {
-            LOGGER.warn("RelayState has invalid size: {}", relayState.getBytes().length);
+        int relayStatelength = relayState.getBytes(StandardCharsets.UTF_8).length;
```

Comment:
Since we're using UTF-8 encoding, the number of bytes could vary based upon the characters contained in the string. If all we want to know is if the string contains 80 characters, shouldn't we just use `String.length()` instead? Unless we want to make sure that the encoded string fits within 80 bytes.